### DISCUSSION TITLE
CNCF TAG-Runtime Discussion/Presentation?

### DISCUSSION THREAD

**Discussion by raravena80** [wasmerio/wasmer#2626] [General]
### Summary

Hello Wasmer team,

I'm one of the co-chairs of the CNCF SIG-Runtime, I'm reaching out and I think it would be great for you to present/discuss the project at one of our meetings. For example, discuss things such as architecture overview and use cases.

Let me know if this something you'd be interested in doing. If yes, please feel free to add it to our [agenda](https://bit.ly/cncf-sig-runtime-meeting-notes) or reach out to me (raravena80 at gmail.com)

Thanks!

### Additional details

N/A

**Comment by syrusakbary** [comment]
Just followed-up via email

**Comment by raravena80** [comment]
Hi, @syrusakbary followed up on email.

Let me know if you would like to present at our next meeting.

Thanks!

**Comment by raravena80** [comment]
@Hywan we never got a reply from this. Is the team still interested in presenting/joining our meetings? Thx.

**Comment by Hywan** [comment]
Sure! I've pinged @syrusakbary again.

**Comment by raravena80** [comment]
@Hywan @syrusakbary following up. Would any of the maintainers be interested in presenting/engager with the CNCF TAG-Runtime?  Thanks.

**Reply by raravena80** [reply]
@Hywan @syrusakbary revisiting here. We'd be happy to hear about the project in the CNCF.
